packages/dashboard/src/dashboard-actions/update.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { update } from './update';
+import { dashboardConfig } from '../testing/mocks';
+import { DashboardState } from '../types';
+
+const state: DashboardState = {
+  dashboardConfiguration: dashboardConfig,
+  intermediateDashboardConfiguration: undefined,
+  selectedWidgetIds: [],
+  numTimesCopyGroupHasBeenPasted: 0,
+  copyGroup: [],
+  stretchToFit: false,
+  width: 1000,
+  cellSize: 10,
+  undoQueue: [],
+  redoQueue: [],
+  previousPosition: undefined,
+};
+
+it('updates dashboard width', () => {
```

Comment:
i'd like to see a test with recursive merging

Reply:
What's recursive merging?